### ISSUE TITLE
Opening stata kernel in notebooks closes existing stata

### ISSUE THREAD

**Issue by jeremysze** [kylebarron/stata_kernel#248]
#### Problem description

Hi

Everytime I open a stata kernel notebook while having an existing Stata and dofile open and running. It closes the Stata window. I'm not sure why that is the case, as I did not find others complaining about the issue. I followed the instructions on your page to install stata kernel.

#### Other information

If you didn't attach the debugging log, please provide:

- Windows 10
- Stata 15.1
- stata-kernel 1.5.9

**Comment by kylebarron** [comment]
Yes, this is known behavior. The issue is that I don't get to choose which Stata session I connect to. By default, it connects to the earliest opened Stata session, or it creates one if it doesn't exist. Then it hides the Stata GUI. The options are:

- The existing behavior
- I can set it so that `stata_kernel` always opens a new Stata GUI window, but then whenever the newly created Stata session closes, any Stata windows that had been opened previously are also closed. I don't know of a way around that.

**Comment by jeremysze** [comment]
Thanks! I think I will just have to keep these issues in mind. Stata kernel has been awesome for sharing work!

**Comment by kylebarron** [comment]
With the next release, with https://github.com/kylebarron/stata_kernel/pull/251, you'll be able to use `%show_gui` to bring up your preexisting Stata session that got hidden.
